stable_baselines3/common/multi_input_envs.py

```diff
@@ -0,0 +1,193 @@
+import gym
+import numpy as np
+
+
+class SimpleMultiObsEnv(gym.Env):
```

Comment:
maybe we should put all custom envs in an env folder...
What do you think @Miffyli ?

Reply:
I agree. Normally a single file (e.g. `envs.py`) would suffice but these classes get lengthy with docstrings etc, so a separate folder `common/envs/` would be nice. We could also consider moving testing environments there so such envs can be used elsewhere as well (local tests with new algos, demonstrations, etc).